src/app/component/eco-news/components/news-list/news-list.component.html

```diff
@@ -47,8 +47,8 @@ <h1 class="main-header">{{ "homepage.eco-news.title" | translate }}</h1>
       </li>
     </ul>
   </div>
-  <app-spinner *ngIf="elementsArePresent; else noNews"></app-spinner>
-  <ng-template #noNews>
+  <app-spinner *ngIf="elementsArePresent"></app-spinner>
+  <ng-template *ngIf="!elementsArePresent && remaining > 0">
```

Comment:
I'm not sure, but maybe will be better to create variable for 0 in .ts file.

Reply:
this is for lack of data, so I think it is not necessary)